lib/nuget-parser/index.ts

```diff
@@ -58,7 +71,11 @@ export async function buildDepTreeFromFiles(
   const tree = {
     dependencies: {},
     meta: {},
-    name: path.basename(root || projectRootFolder),
+    name: getRootName(
+      root,
```

Comment:
and safeRoot instead of root?

Reply:
nope - we need to use the same as before, otherwise existing project names could change unexpectedly